src/Indiko.Maui.Controls.Markdown/MarkdownView.cs

```diff
@@ -353,6 +353,25 @@ public Thickness ListIndent
         set => SetValue(ListIndentProperty, value);
     }
 
+    public static readonly BindableProperty ParagraphSpacingProperty = BindableProperty.Create(nameof(ParagraphSpacing),
+        typeof(double), typeof(MarkdownView), propertyChanged: OnMarkdownTextChanged, defaultValue: 3.0);
+
+    public double ParagraphSpacing
+    {
+        get => (double)GetValue(ParagraphSpacingProperty);
+        set => SetValue(ParagraphSpacingProperty, value);
+    }
+
+    public static readonly BindableProperty LineHeightMultiplierProperty =
```

Comment:
Docs

Reply:
Fixed in https://github.com/0xc3u/Indiko.Maui.Controls.Markdown/pull/23